documentation/platyps-help.md

```diff
@@ -0,0 +1,227 @@
+
+Running the following command will display the help for **all** of your cmdlets:
+```powershell
+Get-HelpPreview -Path $path
+```
+where $path is the path to the MAML help file you generated.
+
+To get the help for a single cmdlet, run the following command:
+```powershell
+$help = Get-HelpPreview -Path $path
+$help | Where-Object {  $_.Name -eq $cmdlet }
+```
+
+## Getting Started
+
+If your service does not currently have any markdown help, follow the below steps to get started.
+
+### Running the `GenerateHelp.ps1` script
+
+If you want to automate this process, run the `tools\platyPS\GenerateHelp.ps1` script with the following command:
+```powershell
+.\GenerateHelp.ps1 -Service $service -PathToRepo $pathToRepo
+```
+where `$service` is the name of your service (*e.g.,* Profile, Compute, Network, etc.) and `$pathToRepo` is the path to the **azure-powershell** repo locally on your machine.
+
+This script can be used for new services looking to create markdown files, or services looking to update their existing markdown files.
+
+### Steps for creating help
+
+**1: Create a folder for your service**
+
+In the `azure-powershell` repository, navigate to the `help` folder in the root, and, if your service does not already have one, create a new folder that will hold the markdown files for your service
+
+**2: Import your module**
+
+Run the following command in PowerShell:
+
+```powershell
+Import-Module "PATH_TO_REPO\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.SERVICE\AzureRM.SERVICE.psd1"
+```
+
+**3: Generate the markdown files**
+
+Run the following command in PowerShell:
+
+```powershell
+New-MarkdownHelp -Module AzureRM.SERVICE -OutputFolder "PATH_TO_REPO\help\SERVICE" -WithModulePage
+```
+
+This will generate a markdown file for each cmdlet in your MAML file. Make sure that all of the cmdlets that you have help for in the MAML file have a corresponding markdown file in the folder.
+
+**4: Generate the MAML file**
+
+Run the following command in PowerShell:
+
+```powershell
+New-ExternalHelp -Path "PATH_TO_REPO\help\SERVICE" -OutputPath "PATH_TO_REPO\src\ResourceManager\SERVICE\Commands.SERVICE -Force"
+```
+
+This will generate a MAML file based off of the markdown files that you created in the previous step and replace the old MAML file.
+
+## Changes to Cmdlets
+
+The following changes to a cmdlet require that the corresponding markdown help file be updated:
+
+- Adding/removing parameters or parameter sets
+- Adding/removing cmdlets
+- Changing metadata of the cmdlet, parameters, or parameter sets
+
+In order to update the markdown help file with the corresponding changes, build the project containing your changes (you may have to close your current PowerShell session to build).
```

Comment:
Put these items in a list

- build the project
- open a new powershell window and import your project, ...